app/src/main/java/org/wikipedia/diff/ArticleEditDetailsFragment.kt

```diff
@@ -212,7 +222,7 @@ class ArticleEditDetailsFragment : Fragment(), WatchlistExpiryDialog.Callback, L
             if (it is Resource.Success) {
                 setLoadingState()
                 viewModel.getRevisionDetails(it.data.rollback?.revision ?: 0)
-                showRollbackSnackbar()
+                showRollbackSnackbar(it.data.rollback?.revision ?: 0)
```

Comment:
Would it be possible to log the action here instead of passing the revision id to the snackbar function?